subiquity/server/controllers/ad.py

```diff
@@ -77,6 +100,15 @@ def __init__(self, app):
         else:
             self.ping_strgy = DcPingStrategy()
 
+    def start(self):
+        if self.ping_strgy.has_support():
+            asyncio.create_task(self._try_discover_domain())
```

Comment:
Bare create_task should no longer be done, see the note marked "important" at https://docs.python.org/3/library/asyncio-task.html#asyncio.create_task 

`run_bg_task` is available to mitigate this, otherwise keep a reference here

Reply:
Thanks for pointing that out!